src/constants/ContentSectionAnchors.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2022 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import { SectionKind } from '@/constants/PrimaryContentSection';
+
+export const MainContentSectionAnchors = {
```

Comment:
I'm not sure right now of a better name off the top of my head, but I think it's a little confusing to have distinguished groupings called "main" and "primary" since they both imply similar meanings.

Reply:
Right... I dont have a better suggestion either, but they are from different places in the RenderJSON, so I did not want to merge them.

Reply:
Keeping them split makes sense. Yeah, not coming up with a better alternative at the moment, but it may be confusing to people.